### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by slavingia**
Using most frequent questions by newcomers to better inform our README.

Also, linking to the Trello which we'll use to manage collection of voter files from each state.

### CODE REVIEW

**Comment by slavingia** [comment]
@JoshSmith @BenGalewsky ready for review (sorry, i forgot which tag to add, i thought it was "ready" but that seems to mean "ready for anyone to tackle" on waffle).

File: README.md [hunk 1]
```diff
@@ -15,8 +15,16 @@ We want to pull politics into the 21st century, and we're starting from the grou
...
 * First, join us on [Slack](http://goo.gl/forms/8SJRDlo7Lx2rUsan1).
-* Then, take a look at [our todo list](http://waffle.io/getmovement/national-voter-file), and see where you'd like to contribute.
-* Finally, learn how to [view and modify the relational model itself](https://github.com/getmovement/national-voter-file/tree/master/docker), the core of our work.
+* Take a look at [our todo list](http://waffle.io/getmovement/national-voter-file).
+* Learn about [the code](https://github.com/getmovement/national-voter-file/tree/master/docker) itself!
+
+Here's the bulk of the work:
+
+1. [Collect voter files for every state](https://trello.com/b/IlZkwYc0/national-voter-file-states-pipeline) and store them securely.
+2. Write state-specific Python scripts to turn them into a standardized input file (SIF).
```
Comment (JoshSmith) [inline README.md]: Is there somewhere we can dive into what an SIF is more? Just link out somewhere?

File: README.md [hunk 2]
```diff
@@ -15,8 +15,16 @@ We want to pull politics into the 21st century, and we're starting from the grou
...
+3. Make sure that data is clean and consistent nationally.
+4. Load that data into a queryable database using Pentaho.
```
Comment (JoshSmith) [inline README.md]: Maybe explicate Pentaho a bit more here?

**Comment by slavingia** [comment]
added pr feedback as separate comment @JoshSmith

**Comment by slavingia** [comment]
commit*

**Pull request merged**